runtime/gc_glue_java/ObjectModel.cpp

```diff
@@ -98,6 +98,7 @@ GC_ObjectModel::internalClassLoadHook(J9HookInterface** hook, UDATA eventNum, vo
 	GC_ObjectModel *objectModel = (GC_ObjectModel*)userData;
 	J9VMThread *vmThread = classLoadEvent->currentThread;
 	J9Class *clazz = classLoadEvent->clazz;
+	clazz->hotFieldsInfo = NULL;
```

Comment:
j9class init should be done in createramclass, also the ram class fragments are zero init so the `= NULL` is not needed.